apps/builder/app/canvas/features/webstudio-component/webstudio-component.tsx

```diff
@@ -117,8 +120,8 @@ export const WebstudioComponentDev = ({
     [componentAttribute]: instance.component,
     [idAttribute]: instance.id,
     onClick: (event: MouseEvent) => {
-      if ((event.target as HTMLElement).closest("a")) {
-        event.preventDefault();
+      if (event.currentTarget instanceof HTMLAnchorElement) {
+        handleLinkClick(event);
```

Comment:
Does it still work with bold inside link?

Reply:
Yeah. 

The reason I've switched from `.closest("a")` is because when you click on "bold" in `Body > Link > Bold` we get 3 events: 

1. `{currentTarger: Body, target: Bold}`
2. `{currentTarger: Link, target: Bold}` — this one we want to preventDefault
3.  `{currentTarger: Bold, target: Bold}`

And all 3 satisfy `event.target.closes('a')`

Reply:
Makes more sense to me to check in general onClick and then call specific handler. Maybe we'll have `handleButtonClick` later or something.

If I move check to `handleLinkClick` then and call it always, then it will be weird that it's called `handleLinkClick` and not just `handleClick`, and if I rename to `handleClick` then semantics/intent will be confusing.